modules/internal/ChapelBase.chpl

```diff
@@ -1496,8 +1496,11 @@ module ChapelBase {
   inline proc >(a: uint(64), param b: uint(64)) {
     return __primitive(">", a, b);
   }
+  inline proc >(param a: uint(?w), b: uint(w)) param where a == 0 {
```

Comment:
I did this somewhat conservatively, concerned that if we were comparing 8-bit values, the compiler might dispatch to the non-param versions and not fold the conditionals.  But granted, that was essentially paranoia.

Looking at the diff above with fresh eyes, it looks to me like we should remove the >(non-param, param) overload that doesn't fold since it doesn't make use of the param-ness of b and is presumably redundant with the >(non-param, non-param version).  Do you see any reason to keep it?